charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -15,7 +15,7 @@ image:
   # -- Victoria Metrics anomaly Docker repository and image name
   repository: victoriametrics/vmanomaly
   # -- Tag of Docker image
-  tag: v1.15.4 # rewrites Chart.AppVersion
+  tag: v1.15.9 # rewrites Chart.AppVersion
```

Comment:
nit: should we leave this null to use `Chart.AppVersion`?

Reply:
vmanomaly should change their tagging policy and start using `-enterprise` suffix, till that moment tag should be set explicitly